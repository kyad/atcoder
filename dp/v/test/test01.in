5 100
1 2
1 3
2 4
2 5
