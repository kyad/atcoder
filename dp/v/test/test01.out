10
12
6
7
7
